Library/Homebrew/test/test_dependency_collector.rb

```diff
@@ -76,6 +76,20 @@ def test_x11_min_version_and_tag
     assert_predicate dep, :optional?
   end
 
+  def test_ant_dep_mavericks_or_newer
+    skip "Only for Mac OS" unless OS.mac?
```

Comment:
This line (and the similar one in the second test) is what I expect to generate the most contention.

Without it, you get: `NameError: uninitialized constant DependencyCollectorTests::MacOS.`

It's not like these two tests make any sense on Linux, but I did add them at the same time of the original PR.